tests/bdd/common/nvme.py

```diff
@@ -69,6 +69,7 @@ def nvme_connect(uri):
     hostnqn = nvme_hostnqn_arg(u)
 
     command = f"sudo nvme connect -t tcp -s {port} -a {host} -n {nqn} {hostnqn}"
+    print(command)
```

Comment:
Do we need this print?

Reply:
It's just a helper in case things fail, nothing gets printed when running pytest with default args.